app/views/account/OAuthAIYouthView.coffee

```diff
@@ -0,0 +1,37 @@
+require('app/styles/account/oauth-aiyouth-view')
+RootView = require 'views/core/RootView'
+template = require 'templates/account/oauth-aiyouth-view'
+utils = require 'core/utils'
+User = require 'models/User'
+
+module.exports = class OAuthAIYouthView extends RootView
+  id: 'oauth-aiyouth-view'
+  template: template
+
+  events:
+    'click .confirm-btn': 'onClickConfirmAuth'
+    'click .change-btn': 'onClickChangeAccount'
+
+
+  initialize: ->
+    @logoutRedirectURL = false
+    @supermodel.trackRequest(me.fetchIsAIYouthBinded())
+    @username = if me.get('firstName') then me.get('firstName') else me.get('name')
```

Comment:
Prefer `name` to `firstName`?

`@username = me.get('name') or me.get('firstName')`

We could also use `me.broadName()` to cover this automatically

Reply:
Alright, we should update broadName logic in China, but didn't do that yet..